server/src/com/mirth/connect/server/userutil/DestinationSet.java

```diff
@@ -24,9 +28,9 @@
  * Utility class used in the preprocessor or source filter/transformer to prevent the message from
  * being sent to specific destinations.
  */
-public class DestinationSet {
+public class DestinationSet implements Set<Integer> {
 
-    private Map<String, Integer> destinationIdMap;
+    private Map<String, Integer> destinationIdMap = Collections.emptyMap();
     private Set<Integer> metaDataIds;
```

Comment:
Should this also default to Collections.emptySet() ?

Reply:
No, as we use its add() which emptySet() doesn't implement